projects/capstone/build.sh

```diff
@@ -34,7 +34,7 @@ do
     export CFLAGS=""
     python setup.py install
     )
-    cd ../suite
+    cd ../../suite
```

Comment:
nit: something like `$SRC/capstone/suite` or whatever an absolute path looks like might be slightly better